hs_geo_raster_resource/models.py

```diff
@@ -50,9 +50,9 @@ def create(cls, **kwargs):
                               if k in ('units', 'northlimit', 'eastlimit', 'southlimit', 'westlimit', 'projection')}
```

Comment:
add a docstring to create() and explain why this class needs to override the create() from its base